mypyc/common.py

```diff
@@ -1,3 +1,5 @@
 PREFIX = 'CPyPy_'
 NATIVE_PREFIX = 'CPyDef_'
 REG_PREFIX = 'cpy_r_'
+
+MAX_62_BIT = (1 << 62) - 1
```

Comment:
I'd rename this to `MAX_SHORT_INT` or something along those lines. The fact that it is 62 bits is an implementation detail, and will be different if we support 32-bit targets.

Reply:
Done.